docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,348 @@
+```
+peer version
+```
+
+You also need to set the `CORE_PEER_MSPCONFIGPATH` to the location of the MSP folder of an admin or client user. Run the following command to set the identity to the Org1 admin user.
+```
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp
+```
+
+You can now create the chaincode package using the `peer lifecycle chaincode package` command:
+```
+peer lifecycle chaincode package fabcar.tar.gz --path ../chaincode/fabcar/go/ --lang golang --label fabcar_1
+```
+
+This command will create a package named ``fabcar.tar.gz`` in your current directory. The `--lang` flag is used to specify the chaincode language and the `--path` flag provides the location of your smart contract code. The `--label` flag is used to specify a chaincode label that will identity your chaincode after it is installed. It is recommended that your label include the chaincode name and version.
+
+Now that we created the chaincode package, we can [install the chaincode](#install-the-chaincode-package) on the peers of the test network.
+
+### JavaScript
+
+### Java
+
+## Install the chaincode package
+
+After we package the Fabcar smart contract, we can install the chaincode on our peers. The chaincode needs to be installed on every peer that will endorse a transaction. Because we are going to set the endorsement policy to require endorsements from both Org1 and Org2, we need to install the chaincode on the peers operated by both organizations:
+
+- peer0.org1.example.com
+- peer0.org2.example.com
+
+Let's install the chaincode on the Org1 peer first. Set the following environment variables to operate the `peer` CLI as the Org1 admin user. The `CORE_PEER_ADDRESS` will be set to point to the Org1 peer, `peer0.org1.example.com`.
+```
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_LOCALMSPID="Org1MSP"
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp
+export CORE_PEER_ADDRESS=localhost:7051
+```
+
+Issue the following command to install the chaincode on the peer:
+```
+peer lifecycle chaincode install fabcar.tar.gz
+```
+
+A successful install command will return the chaincode package identifier. You should see output similar to the following:
+```
+2020-02-12 11:40:02.923 EST [cli.lifecycle.chaincode] submitInstallProposal -> INFO 001 Installed remotely: response:<status:200 payload:"\nIfabcar_1:69de748301770f6ef64b42aa6bb6cb291df20aa39542c3ef94008615704007f3\022\010fabcar_1" >
+2020-02-12 11:40:02.925 EST [cli.lifecycle.chaincode] submitInstallProposal -> INFO 002 Chaincode code package identifier: fabcar_1:69de748301770f6ef64b42aa6bb6cb291df20aa39542c3ef94008615704007f3
```

Comment:
Is it worth mentioning that when the chaincode is installed on the peer that a package identifier is generated and this identifier is used later when the cc is approved.

Also, is the package id unique for each peer where the cc is installed?

Reply:
- If the command is successful, the peer will generate and return the package identifier. This packer ID will be used to approve the chaincode in the next step. 
- The package ID is the combination of the chaincode label and a hash of the chaincode bytes. Every peer will generate the same package ID.

Reply:
Typo: `This packer ID` although I kinda like that slang :-)